resources/route53_domains.go

```diff
@@ -0,0 +1,492 @@
+package resources
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/route53domains"
+	"github.com/aws/aws-sdk-go-v2/service/route53domains/types"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+func Route53Domains() *schema.Table {
+	return &schema.Table{
+		Name:         "aws_route53_domains",
+		Description:  "The domain names registered with Amazon Route 53.",
+		Resolver:     fetchRoute53Domains,
+		Multiplex:    client.AccountRegionMultiplex,
```

Comment:
If I am not incorrect only us-east-1 works on route53domains service. https://github.com/aws/aws-cli/issues/1354 and see https://docs.aws.amazon.com/general/latest/gr/r53.html

Reply:
Good catch, thanks!

Updated to use only the default region.